docs/Extensions.md

```diff
@@ -37,6 +37,58 @@ You can see the [vscode-xml-maven](https://github.com/angelozerr/vscode-xml-mave
 
 ![demo of vscode xml maven suggesting different scopes for a dependency](./images/vscode-xml-maven.gif)
 
+### Debugging your extension
+
+#### VSCode
+
+1. Copy [LemMinX's `.vscode/launch.json`](https://github.com/eclipse/lemminx/blob/master/.vscode/launch.json) into your extension project.
+   The folder structure should look something like this:
+   ```
+   | v .vscode
+   | | - launch.json
+   | v src
+   | | > main
+   | | > test
+   | > target
+   | - pom.xml
+   | - mvnw
+   |   ...
+   ```
+2. In `launch.json`, change the `projectName` to the name of your extension.
```

Comment:
I think it should be more easy to set the content of launch.json (with the custom project name) here instead of saying copy/paste the launch from lemminx.